components/elm/src/biogeochem/AllocationMod.F90

```diff
@@ -1200,6 +1201,7 @@ subroutine Allocation2_ResolveNPLimit (bounds, num_soilc, filter_soilc  , &
                      if (veg_pp%active(p).and. (veg_pp%itype(p) .ne. noveg)) then
                         e_km_nh4 = e_km_nh4 + e_plant_scalar*frootc(p)*froot_prof(p,j)*veg_pp%wtcol(p)/km_plant_nh4(ivt(p))
                         e_km_no3 = e_km_no3 + e_plant_scalar*frootc(p)*froot_prof(p,j)*veg_pp%wtcol(p)/km_plant_no3(ivt(p))
+                        decompmicc(c,j) = decompmicc(c,j) + decompmicc_patch_vr(ivt(p),j)*veg_pp%wtcol(p)
```

Comment:
@qzhu-lbl @rgknox It appears this has already been fixed in master. Correct?

Reply:
yes, already in